README.md

```diff
@@ -10,6 +10,104 @@ Centos 6
 
 Centos 7?
 
+## Usage
+
+### Resource
+
+```
+    python_webapp "python_app_name" do
+      option "option_value"
+    end
+```
+
+Option                  | Type      | Description                                   | Default Value
+----------------------- | --------- | --------------------------------------------- | -------------
+`:create_user`          | Boolean   | Creates a user on the sever                   | `false`
+`:path`                 | String/Nil| Path to application                           | `/opt/#{ project_name }`
+`:owner`                | String    | Owner of application files (user)             | `chef`
+`:group`                | String    | Owner of application files (group)            | `chef`
+`:repository`           | String    | URL to git repository                         | `nil`
+`:revision`             | String    | Branch or Commit Hash of Repo                 | `master`
+`:virtualenv_path`      | String    | Path to python virtualenv                     | `/opt/venv_#{ project_name }`
+`:config_template`      | String/Nil| Path to app config template                   | `settings.py.erb`
+`:config_destination`   | String/Nil| Path to app config location on server         | `nil`
+`:config_vars`          | Hash      | Variables for the config template             | `nil`
+`:requirements_file`    | String/Nil| Application's dependencies file.              | `setup.py`
+`:django_migrate`       | Boolean   | Whether or not to run django migrations       | `false`
+`:django_collectstatic` | Boolean   | Whether or not to run django collectstatic    | `false`
+`:interpreter`          | String    | Python command (python, python3, etc)         | `python`
+`:gunicorn_port`        | Int/Nil   | Port to run gunicorn on                       | `nil`
```

Comment:
It'd be nice to know what default behavior happens when things are left as `Nil`. In some cases, we pick a sane default, and in other cases we just don't include that optional thing.